### PR TITLE
docs: add instruction for downloading logs

### PR DESCRIPTION
**Pull request by lastoCHka42**
https://docs.d.tarantool.io/en/dev/multivac/backup/

### CODE REVIEW

File: docs/backup.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+        $  aws s3 --endpoint-url https://hb.bizmrg.com \
+            ls s3://multivac/tarantool/tarantool
+
+4.  Download logs from s3
+
+    ..  code-block:: console
+
+        $ aws s3 --endpoint-url https://hb.bizmrg.com \
+            sync s3://multivac/tarantool/tarantool/workflow_run_jobs/ your_path
```
Comment (Serpentian) [inline docs/backup.rst:30]: In general, instruction looks great to me.

But don't we want to mention at least approximately the size of the `workflow_run_jobs` directory? The user might not have enough space on his disc to download all the logs. According to NickVolynkin it's something about 50Gb.

File: docs/backup.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+========================
+
+1.  Install aws according to
+    `official instruction <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`__.
+
+2.  Set up aws
+
+    ..  code-block:: console
+
+        $ aws configure
```
Comment (Serpentian) [inline docs/backup.rst:11]: Is it a good practice to recommend configuring the default profile? Maybe it's better to specify it:
```sh
$ aws configure --profile any_profile_name
```

Commands  below should also be changed if we decide to specify the profile.

File: docs/backup.rst (hunk 1 above)
Comment (Serpentian) [reply docs/backup.rst:34]: Actually it's 123Gb now

File: docs/backup.rst (hunk 2 above)
Comment (Serpentian) [reply docs/backup.rst]: All of the commands should be changed. e.g.:

```
 $  aws s3 --endpoint-url https://hb.bizmrg.com --profile any_profile_name \
            ls s3://multivac/tarantool/tarantool
``` 

If we don't specify the profile, default one is used.

I don't really insist on that change. Let's either use default one, or specify it with `--profile` everywhere.

Comment (lastoCHka42) [reply docs/backup.rst]: Fixed

File: docs/backup.rst (hunk 1 above)
Comment (lastoCHka42) [reply docs/backup.rst:34]: Fixed

**Review by Serpentian — APPROVED** [review]
Nice work! LGTM

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**